pkg/controller/v1beta1/inferenceservice/utils/utils.go

```diff
@@ -18,9 +18,22 @@ package utils
 
 import (
 	v1beta1api "github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var (
+	log = logf.Log.WithName("inferenceservice-v1beta1--utils")
 )
 
 // Only enable MMS predictor when predictor config sets MMS to true and storage uri is not set
-func IsMMSPredictor(predictor *v1beta1api.PredictorSpec, isvcConfig *v1beta1api.InferenceServicesConfig) bool {
-	return predictor.GetImplementation().IsMMS(isvcConfig) && predictor.GetImplementation().GetStorageUri() == nil
+func IsMMSPredictor(isvc *v1beta1api.InferenceService, isvcConfig *v1beta1api.InferenceServicesConfig) bool {
+	predictor := isvc.Spec.Predictor
+	isMMS := predictor.GetImplementation().IsMMS(isvcConfig) && predictor.GetImplementation().GetStorageUri() == nil
+
+	if isMMS {
+		log.Info("Predictor is configured for multi-model serving", "InferenceService", isvc.Name)
```

Comment:
this might be a bit excessive for info level, do we need this ?

Reply:
I'm debating on whether or not it's better to have: `log.V(5).Info("Determining if predictor is configured for multi-model serving", "InferenceService", isvc.Name, "IsMMS", isMMS)` rather than the condition statements. What do you think?